src/app/pages/vm/vm-list/delete-vm-dialog/delete-vm-dialog.component.html

```diff
@@ -0,0 +1,36 @@
+<h1 mat-dialog-title>{{ 'Delete Virtual Machine' | translate }}</h1>
+<div mat-dialog-content>
+  <form [formGroup]="form">
+    <ix-checkbox
+      formControlName="zvols"
+      [label]="'Delete Virtual Machine Data?' | translate"
+      [tooltip]="helptext.delete_dialog.zvolsTooltip | translate"
+    ></ix-checkbox>
+    <ix-checkbox
+      formControlName="force"
+      [label]="'Force Delete?' | translate"
+      [tooltip]="helptext.delete_dialog.forceTooltip | translate"
+    ></ix-checkbox>
+    <ix-input
+      formControlName="confirmName"
+      [label]="'Enter {name} below to confirm.' | translate: { name: vm.name }"
```

Comment:
There is a similar string that may have translations:
`"Enter {value} below to confirm."`